app/forms/register_user_email_form.rb

```diff
@@ -58,6 +62,14 @@ def valid_form?
     @allow && valid? && !email_taken?
   end
 
+  def set_email_taken
```

Comment:
Can we call this `look_up_email_taken` or something? It doesn't have any side effects, so the `set_` is throwing me

Reply:
Yeah, can do